test/service/recognition.js

```diff
@@ -324,19 +324,40 @@ describe("service/recognition", () => {
   describe("gratitudeReceiverIdsIn", () => {
```

Comment:
Adding some additional tests to cover the group functionality of at least `gratitudeReceiverIdsIn` would be good if possible.